src/Uno.UWP/System/Power/PowerManager.iOS.cs

```diff
@@ -0,0 +1,150 @@
+﻿#if __IOS__
+using System;
+using UwpBatteryStatus = Windows.System.Power.BatteryStatus;
+using UwpEnergySaverStatus = Windows.System.Power.EnergySaverStatus;
+using UwpPowerSupplyStatus = Windows.System.Power.PowerSupplyStatus;
+using UIKit;
+using Foundation;
+
+namespace Windows.System.Power
+{
+	public partial class PowerManager
+	{
+		private const string DeviceModelSimulator = "iPhone Simulator";
+
+
+		private static NSObject _batteryLevelChangeSubscription;
+		private static NSObject _batteryStateChangeSubscription;
+		private static NSObject _powerStateChangeSubscription;
+
+		private static readonly UIDevice _device;
+		private static readonly bool _isSimulator;
+
+		static PowerManager()
+		{
+			_device = UIDevice.CurrentDevice;
+			_isSimulator = _device.Model == DeviceModelSimulator;
```

Comment:
Hmm what does `Model` return if you launch on a simulated iPad?

Reply:
I originally found a resource that said there is only iPhone Simulator value returned, but now I found otherwise. Will fix